ci/release/template/man/d2.1

```diff
@@ -80,6 +80,9 @@ Renders the diagram to look like it was sketched by hand
 .It Fl -center Ar flag
 Center the SVG in the containing viewbox, such as your browser screen
 .Ns .
+.It Fl -scale Ar -1
+Set a fixed render scale (e.g. 0.5 to halve rendered size). If none provided, SVG will fit to screen
```

Comment:
"Scale the output. E.g., 0.5 to halve the default size. Default -1 means that SVG's will fit to screen and all others will use their default render size. Setting to 1 turns off SVG fitting to screen."